### PR TITLE
Fixes!

### LINKED ISSUE #27: Input on lobby not working in sample project
In the demo project: https://docs.joinplayroom.com/demos/unity/index.html

The unity Input system is capturing the keystrokes and not sending them to input box. Can we fix this by not starting input capture on unity until insertCoin callback is called, in example project.

### PR DESCRIPTION
**Pull request by momintlh**
* Fixed #27 
- [x] Tested by Playroom team
- [ ] Tested by users
* Tested SetState<Bool>() and GetState<Bool>(), working fine based on #26
- [x] Tested by Playroom team
- [ ] Tested by users
* Added cross-origin #25
- [x] Tested by Playroom team
- [ ] Tested by users
* Removed logs #28
- [x] Tested by Playroom team
* Added the CultureInfo.InvariantCulture to SetStateFloat #23  
- [x] Tested by Playroom team
- [ ] Tested by users

Did a build test on the demo game.

### CODE REVIEW

**Comment by SaadBazaz** [comment]
Please share a test report. We can't afford feature degradation rn.

File: Assets/Plugins/PlayroomPlugin.jslib [hunk 1]
```diff
@@ -23,7 +24,6 @@ mergeInto(LibraryManager.library, {
     }
 
     var options = optionsJson ? JSON.parse(UTF8ToString(optionsJson)) : {};
-    // console.log(options)
```
Comment (SaadBazaz) [inline Assets/Plugins/PlayroomPlugin.jslib:26]: Remove, don't comment.
Keeps the code neat and clean.

File: Assets/PlayroomKit/PlayroomKit.cs [hunk 2]
```diff
@@ -929,7 +929,7 @@ public void SetState(string key, float value, bool reliable = false)
             {
                 if (IsRunningInBrowser())
                 {
-                    SetPlayerStateFloatByPlayerId(id, key, value.ToString(), reliable);
+                    SetPlayerStateFloatByPlayerId(id, key, value.ToString(CultureInfo.InvariantCulture), reliable);
```
Comment (SaadBazaz) [inline Assets/PlayroomKit/PlayroomKit.cs:932]: Is this done everywhere where `ToString` is used? If yes, has it been tested to not have any negative effects?

**Review by SaadBazaz — APPROVED** [review]
Approved, with suggestions.

File: Assets/Plugins/PlayroomPlugin.jslib [hunk 3]
```diff
@@ -12,6 +12,7 @@ mergeInto(LibraryManager.library, {
     // onLaunchCallback
   ) {
     function embedScript(src) {
+      script.crossOrigin = 'anonymous';
```
Comment (asadm) [inline Assets/Plugins/PlayroomPlugin.jslib:15]: tested this? does this cause any issues?

**Review by asadm — APPROVED** [review]

File: Assets/Plugins/PlayroomPlugin.jslib (hunk 3 above)
Comment (SaadBazaz) [reply Assets/Plugins/PlayroomPlugin.jslib:15]: Requested and tested by a user here #25. 
Does not break the SDK in any way, however, the security of it is debatable. We shouldn't allow **all** origins I suppose.
Will keep this under watch.

**Pull request merged**